src/cbc_syslog/util/carbon_black_cloud.py

```diff
@@ -13,7 +13,7 @@
 """Carbon Black Cloud class"""
 
 from cbc_sdk import CBCloudAPI
-from cbc_sdk.platform import BaseAlert  # Re-add when CBC SDK is released AuditLog
+from cbc_sdk.platform import Alert, AuditLog
```

Comment:
Really not related to this PR, but can these instructions (or a variation) please be added in the readme?

https://community.carbonblack.com/t5/Developer-Relations/How-to-Create-a-Scheduled-Task-to-Run-your-Python-Scripts/m-p/893